index.html

```diff
@@ -1,20 +1,116 @@
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="css/style.css">
+
+</head>
+
+<body>
+
+  <nav>
+
+    <ul>
+      <li>
+        <a class="logo" href="#">Relaxr</li>
+      </a>
+      <li>
+        <a class="boxmenu" href="file:///Users/lornamichelle/Documents/GitHub/FEWD-Assignment2/index.html">About</li>
+      </a>
+      <li>
+        <a class="boxmenu" href="#">FAQ</li>
+      </a>
+      <li>
+        <a class="boxmenu" href="#">Team</li>
+      </a>
+      <li>
+        <a class="boxmenu" href="#">Contact Us</li>
+      </a>
+      <li>
+        <a class="boxmenu" href="file:///Users/lornamichelle/Documents/GitHub/FEWD-Assignment3/index.html#">Blog</li>
+      </a>
+    </ul>
+
+  </nav>
+
+  <div class="secondarycontainer">
+
+    <div class="box1">
+
+      <h1>How I implemented Relaxr in 2 weeks and </br> changed my life.</h1>
+      <img src="https://raw.githubusercontent.com/lornamicheller/FEWD-Assignment3/master/images/blog_photo1.jpg" alt="people paddling">
+      <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ipsam eum assumenda dolorum modi dolore quas fugit? Facere
+        nemo quos maxime quis numquam quibusdam dignissimos nihil distinctio, maiores veritatis quasi ex! Lorem ipsum dolor
+        sit amet consectetur adipisicing elit. Perspiciatis, magni nemo illo sunt asperiores enim illum veniam quidem error
+        quisquam aut perferendis omnis velit exercitationem expedita modi cum animi.</p>
+      <!-- <h2>Read more > </h2> -->
+
+      <div class="readmore">Read more ></div>
+
+      <div class="divisor"></div>
+
+      <h1>I traveled across the globe thanks to Relaxr's </br> Excel automation.</h1>
+      <img src="https://raw.githubusercontent.com/lornamicheller/FEWD-Assignment3/master/images/blog_photo2.jpg" alt="little boats">
+      <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ipsam eum assumenda dolorum modi dolore quas fugit? Facere
+        nemo quos maxime quis numquam quibusdam dignissimos nihil distinctio, maiores veritatis quasi ex! Lorem ipsum dolor
+        sit amet consectetur adipisicing elit. Perspiciatis, magni nemo illo sunt asperiores enim illum veniam quidem error
+        quisquam aut perferendis omnis velit exercitationem expedita modi cum animi.</p>
+      <!-- <h2>Read more ></h2> -->
+
+      <div class="readmore">Read more ></div>
+
+      <div class="divisor"></div>
 
-  </head>
-  <body>
-    <div class="container">
-      <!-- Replace me with your html -->
     </div>
-  </body>
+
+    <div class="box2">
+
+      <h3>Categories</h3>
+      <ul class="secondarylist">
+        <li>Sucess stories</li>
+        <li>Stats</li>
+        <li>How tos</li>
+        <li>Best of</li>
+        <li>Business tips</li>
+      </ul>
+      <h3>About Relaxr</h3>
+      <p1>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quas in facilis vero, esse cupiditate voluptate voluptatibus.
```

Comment:
<p1> isn't a tag. You can add a class to the <p> tag, but you can't change the name of it.